gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
```

Comment:
I have checked your change locally to run the build by `./gradlew build`.  The first line is still trying to downloaded gradle 4.6, is this expected?

`uncwang@juncwang-ld1 ~/w/kyle_pegasus ��������� git clone https://github.com/DPUkyle/rest.li.git
Cloning into 'rest.li'...
remote: Enumerating objects: 29, done.
remote: Counting objects: 100% (29/29), done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 61652 (delta 9), reused 18 (delta 6), pack-reused 61623
Receiving objects: 100% (61652/61652), 19.97 MiB | 18.36 MiB/s, done.
Resolving deltas: 100% (23896/23896), done.
juncwang@juncwang-ld1 ~/w/kyle_pegasus ��������� 
juncwang@juncwang-ld1 ~/w/kyle_pegasus ��������� 
juncwang@juncwang-ld1 ~/w/kyle_pegasus ��������� 
juncwang@juncwang-ld1 ~/w/kyle_pegasus ��������� ls
rest.li
juncwang@juncwang-ld1 ~/w/kyle_pegasus ��������� cd rest.li
juncwang@juncwang-ld1 ~/w/k/rest.li ��������� ls
build_script  d2-schemas           data-transform  gradle           pegasus-common         r2-jetty       restli-client-parseq         restli-disruptor       restli-guice-bridge        restli-netty-standalone   restli-tools  defaultEnvironment.gradle  prepare-release
buildSrc      data                 degrader        gradle-plugins   r2                     r2-netty       restli-client-testutils      restli-docgen          restli-internal-testutils  restli-server             test-util     ERROR-CODES.md             pre-release-check
d2            data-avro            entity-stream   li-jersey-uri    r2-core                r2-perf-test   restli-client-util-recorder  restli-example-api     restli-int-test            restli-server-extras      tools         gradle.properties          README.md
d2-benchmark  data-avro-1_6        examples        li-protobuf      r2-disruptor           r2-sample      restli-common                restli-example-client  restli-int-test-api        restli-server-standalone  build.gradle  gradlew                    RESTLI-2-RELEASE-NOTES.md
d2-contrib    data-avro-generator  generator       log-test-config  r2-filter-compression  r2-testutils   restli-common-testutils      restli-example-server  restli-int-test-client     restli-server-testutils   CHANGELOG     gradlew.bat                settings.gradle
d2-int-test   data-testutils       generator-test  multipart-mime   r2-int-test            restli-client  restli-contrib-spring        restli-extras          restli-int-test-server     restli-spring-bridge      checkout-tag  LICENSE.TXT                URI.md
juncwang@juncwang-ld1 ~/w/k/rest.li ��������� ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-4.6-bin.zip
......................................................................
Unzipping /home/juncwang/.gradle/wrapper/dists/gradle-4.6-bin/4jp4stjndanmxuerzfseyb6wo/gradle-4.6-bin.zip to /home/juncwang/.gradle/wrapper/dists/gradle-4.6-bin/4jp4stjndanmxuerzfseyb6wo
Set executable permissions for: /home/juncwang/.gradle/wrapper/dists/gradle-4.6-bin/4jp4stjndanmxuerzfseyb6wo/gradle-4.6/bin/gradle
Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details
Parallel execution with configuration on demand is an incubating feature.
Generating JAR file 'gradle-api-4.6.jar'

`